src/translations/installer_ru.ts

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ru_RU">
+<context>
+    <name>WindowsInstaller</name>
+    <message>
+        <source>__LANGUAGE_NAME__</source>
+        <translatorcomment>This must be the name of one of the NSIS translation files at https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Contrib/Language%20files/ without the file extension.</translatorcomment>
+        <translation>Russian</translation>
+    </message>
+    <message>
+        <source>BirdtraySectionDescription</source>
+        <translation>Бесплатное уведомление в системном трее о новой почте для Thunderbird.</translation>
+    </message>
+    <message>
+        <source>WinIntegrationSectionName</source>
+        <translation>Интеграция с Windows</translation>
+    </message>
+    <message>
+        <source>WinIntegrationGroupDescription</source>
+        <translation>Выберите, как интегрировать программу в Windows.</translation>
+    </message>
+    <message>
+        <source>AutoRunSectionName</source>
+        <translation>Автозагрузка</translation>
+    </message>
+    <message>
+        <source>AutoRunDescription</source>
+        <translation>Автоматически запускать ${PRODUCT_NAME} после включения.</translation>
+    </message>
+    <message>
+        <source>AutoCheckUpdateSectionName</source>
+        <translation>Автоматическая проверка обновлений</translation>
+    </message>
+    <message>
+        <source>AutoCheckUpdateDescription</source>
+        <translation>Автоматическая проверка обновлений для ${PRODUCT_NAME} при запуске.</translation>
+    </message>
+    <message>
+        <source>ProgramGroupSectionName</source>
+        <translation>Создать программную группу</translation>
+    </message>
+    <message>
+        <source>ProgramGroupDescription</source>
+        <translation>Создана программная группа ${PRODUCT_NAME} в меню «Пуск»&gt; «Программы».</translation>
+    </message>
+    <message>
+        <source>DesktopEntrySectionName</source>
+        <translation>Значок на рабочий стол.</translation>
```

Comment:
```suggestion
        <translation>Значок на рабочий стол</translation>
```
This is also the name of a component in the list and shouldn't have a dot at the end.

Also, could you please comment on @gyunaev suggestion:

> I think its значок, а не значёк